M2/Macaulay2/packages/MinimalPrimes/doc.m2

```diff
@@ -0,0 +1,310 @@
+undocumented { installMinprimes }
```

Comment:
Deprecated routines have to continue to be documented.  If users don't know what the routines do, how can they figure how to stop using them?

Reply:
If they use them, they'll get a warning that says please stop using me.

Reply:
Also, there's a message about its deprecation in changes.m2. That should suffice.

Reply:
No, deprecated routines have to continue to be documented. If users don't know what the routines do, how can they figure how to stop using them?